.mergify.yml

```
@@ -175,3 +175,16 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
+  - name: backport patches to 8.6 branch
+    conditions:
+      - merged
+      - label=backport-v8.6.0
```

We need to figure out how to fix this automation. It messes up the label every time.

```suggestion
      - label=backport-8.6
```